fibbingnode/southbound/fibbing.py

```diff
@@ -323,6 +323,14 @@ def proxy_remove(self, points):
     def proxy_connected(self, session):
         self.root.send_lsdblog_to(session)
 
+        if self.routes:
+            log.info("Removing old routes from previous sessions...")
+            for r in self.routes.iterkeys():
+                try:
+                    self.remove_route(r)
+                except:
```

Comment:
What kind of exception can happen there ?
It would be great if the exception was handled in remove_route directly so you could simplify this whole block as (including the if part):
map(self.remove_route, self.routes.iterkeys())

I suspect that one of the errors would be that the dict size has changed so you would need to change iterkeys() to keys()